Kitodo/src/main/java/org/kitodo/production/forms/LoginForm.java

```diff
@@ -11,21 +11,30 @@
 
 package org.kitodo.production.forms;
 
+import java.io.IOException;
 import java.io.Serializable;
 import java.util.Objects;
 
 import javax.enterprise.context.SessionScoped;
+import javax.faces.context.ExternalContext;
+import javax.faces.context.FacesContext;
 import javax.inject.Named;
 
 import org.kitodo.data.database.beans.User;
+import org.kitodo.data.database.exceptions.DAOException;
+import org.kitodo.data.exceptions.DataException;
+import org.kitodo.production.controller.SessionClientController;
+import org.kitodo.production.security.CustomLoginSuccessHandler;
 import org.kitodo.production.services.ServiceManager;
+import org.primefaces.PrimeFaces;
 
 @Named("LoginForm")
 @SessionScoped
 public class LoginForm implements Serializable {
     private User loggedUser;
-    private boolean alreadyLoggedIn = false;
     private boolean firstVisit = true;
+    private static final String INDEXING_PAGE = "system.jsf?tabIndex=2";
+    private static final String DESKTOP_VIEW = "desktop.jsf";
```

Comment:
Since these two are constants the better way is to declare them at the beginning of the class instead of inserting some static Strings into the middle of the code.